lib/Importer/ONNXModelLoader.cpp

```diff
@@ -1199,8 +1199,8 @@ ONNXModelLoader::ONNXModelLoader(const std::string &modelDescFilename,
     RETURN_IF_ERR(setVersion(modelDef));
 
     ONNX_NAMESPACE::GraphProto graphDef = modelDef.graph();
-    RETURN_IF_ERR(checkInputs(graphDef, tensorNames, types));
 
+    RETURN_IF_ERR(checkInputs(graphDef, tensorNames, types));
```

Comment:
Looks like an extra space was inserted above this line. Was this a `clang-format` change? If not, please undo.